code/States/StateHandler.cs

```diff
@@ -31,12 +31,5 @@ public void ChangeState( BaseState state )
 			State = state;
 			State?.Start();
 		}
-
-		[Event.Tick.Server]
-		private void Tick()
-		{
-			Game.Instance.State = State;
-			State?.OnTick();
```

Comment:
I think we still need to be ticking the state, we just don't need to be setting it each tick. I'll merge this in and reimplement the OnTick